python/mxnet/ndarray/numpy/random.py

```diff
@@ -193,6 +194,83 @@ def normal(loc=0.0, scale=1.0, size=None, dtype=None, ctx=None, out=None):
                            ctx=ctx, dtype=dtype, out=out)
 
 
+def logistic(loc=0.0, scale=1.0, size=None):
```

Comment:
Can we support `ctx` as well? Otherwise, GPU implementation is not fully tested.